editor/src/clj/editor/console.clj

```diff
@@ -311,18 +315,33 @@
 
 (defonce ^SimpleStringProperty find-term-property (doto (SimpleStringProperty.) (.setValue "")))
 
+(def ^:private filter-svg-path
+  (ui/load-svg-path "scene/images/filter.svg"))
```

Comment:
`scene/images/`is the only dir that hosts SVG icons for now. It' s probably not a great fit for this case, but I don't know if there is a better option, that does not require broader restructuring.